syft/frameworks/torch/tensors/interpreters/large_precision.py

```diff
@@ -110,6 +106,8 @@ def __iadd__(self, other):
 
         return self
 
+    add_ = __iadd__
```

Comment:
Did you declare `add_` just for the tests?
Actually I think that you can write directly `x. __iadd__(y)` or `x += y`

Reply:
NB actually add_ is a valid torch function